java/src/main/java/com/logicalclocks/hsfs/engine/SparkEngine.java

```diff
@@ -93,6 +95,53 @@ private SparkEngine() {
     sparkSession.conf().set("hive.exec.dynamic.partition.mode", "nonstrict");
     // force Spark to fallback to using the Hive Serde to read Hudi COPY_ON_WRITE tables
     sparkSession.conf().set("spark.sql.hive.convertMetastoreParquet", "false");
+
+    validateSparkConfiguration();
+  }
+
+  @SneakyThrows
+  private void validateSparkConfiguration() {
+    String key = "spark.hadoop.hops.ssl.trustore.name";
+    if (sparkSession.conf().get(key).isEmpty())
+      throw new FeatureStoreException("Invalid: " + key);
```

Comment:
Maybe make the message a bit more informative and define the repeated part only once, e.g.
`"Spark is misconfigured for communication with Hopsworks, missing or invalid property: xyz"`